src/python/pants/core/goals/export_test.py

```diff
@@ -146,14 +146,16 @@ def do_mock_export(req: ExportRequest):
                 DistDir(relpath=Path("dist")),
                 create_subsystem(ExportSubsystem, resolve=resolves, bin=binaries),
             ],
+            mock_calls={
+                "pants.engine.intrinsics.add_prefix": lambda *xs: rule_runner.request(Digest, xs),
+            },
             mock_gets=[
```

Comment:
Should I port all of these unused MochGet's? Despite migrating to call-by-name, run_rule_with_mocks is not complaining about missing mock calls.